pkg/app/app.go

```diff
@@ -982,10 +982,8 @@ func (a *App) visitStatesWithSelectorsAndRemoteSupport(fileOrDir string, converg
 
 	envvals := []interface{}{}
 
-	if a.ValuesFiles != nil {
-		for i := range a.ValuesFiles {
-			envvals = append(envvals, a.ValuesFiles[i])
-		}
+	for _, i := range a.ValuesFiles {
```

Comment:
Nit but I think this should better be `f` or `v`. `i` looks like an index!

Reply:
@mumoshu fix it.